website/docs/docs/dbowser-demo.md

```diff
@@ -0,0 +1,81 @@
+
+Why is modeling this data tricky enough to warrant a tutorial? A few reasons:
+
+- **Volume**: If you’re sending multiple emails per week, tracking what happens with those emails, and have thousands of contacts, that’s thousands of rows generated in a short time span. Modeling data at this volume can be intimidating, expensive, and time-consuming.
+- **Necessity**: This email data should be directly accessible within an email platform and a business user should be able to look at it there, but generally, because of the volume of data, email platforms don’t support the best analytics infrastructure. So, data teams end up needing to do this data modeling out of necessity to bring into their BI tool.
+- **PII**: Emails are sent to real people who have real names, email addresses, and potential physical addresses. Depending on your data team’s governance and PII rules, it’s potentially very important that PII is masked in email data, which inherently adds a layer of complexity during data modeling.
+
+So what are we actually trying to do in this tutorial?
+
+We’ll walk through the common data sources and models required to build up to a final `fct_email_sends` model that captures all email sends, actions, their associated campaign, recipient information, and stats per those dimensions. Your resulting table will ultimately look a little like this:
+
+![Sample  final table](/img/sample_email_data.png)
+
+## Step 1: Explore your sources
+In this tutorial, we have 5 sources from HubSpot, a marketing CRM, that are used to ultimately capture email sends. Below, we’ll breakdown what these 5 source tables are and give you some space to explore this raw data.
+
+- `hubspot__marketing_email_campaign`: The raw table of marketing emails and their connection to campaigns in HubSpot.
+- `hubspot__marketing_email`: The raw table of all marketing emails and their details (content, author, created date, etc.) in HubSpot, not necessarily connected to campaigns.
+- `hubspot__email_event:` A log table of all email events (open/sends/clicks, etc.) with a unique id per email event.
+- `hubspot__email_event_sent`: A log of all emails sent from your HubSpot account.
+- `hubspot__contact`: The list of contacts in your HubSpot account, as well as some relevant information, such as first name and last name.
+
+For the sake of this tutorial, we’re loading in our sources as [seeds](https://docs.getdbt.com/docs/build/seeds), static CSV files dbt can load into your data warehouse. In reality, if you were doing this with your own HubSpot (or other email platform) connector (via some Fivetran, Stitch, or other ETL tool), you would have this raw data already in your data warehouse and would treat them as [sources](https://docs.getdbt.com/docs/build/sources) in dbt.
+
+Use this first IDE to explore these raw data sources, perform some exploratory data analysis (EDA), and get familiar with your dbt project. No wrong answers here 🙂
+
+<dbtEditor project="jaffle_shop" />
```

Comment:
@KiraFuruichi FYI this is how we can call an editor component for a project - the project files live in /static/dbt_projects/project_name, so we'll want to create a separate project there for each of the steps